digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/MassTransitStopDao.scala

```diff
@@ -625,6 +625,10 @@ class MassTransitStopDao {
     query + s" where a.id = $id"
   }
 
+  def withIdAndNotExpired(id: Long)(query: String): String = {
+    query + s" where a.id = $id and (a.valid_to > sysdate or a.valid_to is null)"
+  }
+
```

Comment:
Instead of creating a new method why we don't transform the method above to have the possibility to return only the not expired records, like:
```scala
  def withId(id: Long, withoutExpired: Boolean)(query: String): String = {
    val validToCondition = if (withoutExpired) " and (a.valid_to > sysdate or a.valid_to is null)" else ""
    query + s" where a.id = $id $validToCondition"
  }

Reply:
This should be deleted, not in use anymore since the change of course of the implementation.

Reply:
@marquesfm This part was deleted